yaml/csr.yaml.m4

```diff
@@ -2661,6 +2670,7 @@ ifelse(eval(X_EXT != 0), 1, [[[
       lsb: 21
       warl_legalize: |
         val_out = 0
+]]])
```

Comment:
This clause is okay, but the next one is wrong. If V_EXT != 0, then the V bit will be hardwired to 1 (so WARL (0x1))